examples/src/app/ApplicationAnt.tsx

```diff
@@ -99,6 +100,10 @@ export class ApplicationAnt extends React.Component<{ app?: Application }> {
                                     <NumericFormAnt operation={application.testNumeric} />
                                     <br />
                                     <br />
+                                    <h3>TextAnt - Input Component</h3>
```

Comment:
The text in the header should be: `TagAnt`, this TextAnt name is already in the test page above